tm2/pkg/bft/config/config.go

```diff
@@ -24,6 +25,7 @@ type Config struct {
 	P2P       *p2p.P2PConfig       `toml:"p2p"`
 	Mempool   *mem.MempoolConfig   `toml:"mempool"`
 	Consensus *cns.ConsensusConfig `toml:"consensus"`
+	Indexer   *txindex.Config      `toml:"indexer"`
```

Comment:
It's not indexing though, it's event persisting. 
The indexer is another program that reads the persisted state.

Reply:
EventStore?

Reply:
I completely agree with the reasoning 💯 

I've renamed the "indexer" references to refer to "event store":
[443953d](https://github.com/gnolang/gno/pull/546/commits/443953d0e55302e5aa22ccfe50423b3ede5fb9c6)
and
[10c2483](https://github.com/gnolang/gno/pull/546/commits/10c248305a9f9a45e41446284728642f467d83cb)